packages/kotti-ui/source/next/kotti-field-date/styles.scss

```diff
@@ -159,6 +159,38 @@ $year-font-size: 1rem;
 				}
 			}
 		}
+
+		.in-range {
+			background-color: var(--interactive-04);
+
+			@supports (clip-path: inset(15% -50% 15% -50%)) {
+				clip-path: inset(15% -50% 15% -50%);
+			}
+
+			&.start-date {
+				background: linear-gradient(
+					270deg,
+					var(--interactive-04) 0%,
+					var(--interactive-04) 50%,
+					rgba(0, 0, 0, 0) 50%
+				);
+			}
+			&.end-date {
+				background: linear-gradient(
+					90deg,
+					var(--interactive-04) 0%,
+					var(--interactive-04) 50%,
+					rgba(0, 0, 0, 0) 50%
+				);
+			}
+			&.start-date.end-date {
+				background: transparent !important;
+			}
+		}
+		.start-date:hover,
+		.end-date:hover {
+			background-color: transparent !important;
```

Comment:
please find the part where i add a hover to td:hover and exclude in-range 

please refrain from the !important pattern

Reply:
Done, appears to allow removing all `!important`s.

Reply:
c5711aec7d881cceccac92a53db555db7d098edd